File: Erik_Vincent/Assignment1/Assignment1/Model/QLS/AST/Style/Font.cs
using Assignment1.Model.QL.AST;

namespace Assignment1.Model.QLS.AST.Style
{
    public class Font : ASTNode, IStyle
    {
        public string Value { get; }

        public Font(int lineNumber, string value)
        {
            _lineNumber = lineNumber;
            Value = value;
        }

        public void Accept(IStyleVisitor visitor) => visitor.Visit(this);
    }
}

File: Joanna_Laurence/src/main/java/org/uva/sea/languages/qls/parser/elements/style/Font.java
package org.uva.sea.languages.qls.parser.elements.style;

import org.antlr.v4.runtime.Token;
import org.uva.sea.languages.qls.parser.visitor.IStyleASTVisitor;

public class Font extends StyleSpecification {

    private final String name;

    public Font(final Token token, final String name) {
        super(token);
        this.name = name;
    }

    public final String getName() {
        return this.name;
    }

    @Override
    public final <T> T accept(final IStyleASTVisitor<T> visitor) {
        return visitor.visit(this);
    }
}
